DOCSP-13866 run a command usage example (#24)
mongodb/docs-golang: source/usage-examples/command.txt

.. default-domain:: mongodb

> You can run commands directly on the MongoDB server CLI using the

Reviewer: ```suggestion
You can run commands directly on your MongoDB server using the
```

Question (possible Issue): This sounds odd, as "CLI" implies a Command Line, and I don't think this is executed from a command line (I think the server just runs the command). I see this is also the phrasing in the Java Docs, however it does not seem to be the phrasing in the Node Docs or the [Mongosh docs](https://docs.mongodb.com/manual/reference/method/db.runCommand/). Can this be double checked?

``RunCommand()`` method. 

The following example passes a document containing the ``"dbStats"``

> command to the ``RunCommand()`` method, which requests for statistics

Reviewer: ```suggestion
command to the ``RunCommand()`` method, which retrieves statistics
```
Suggestion: "Requests for" sounds odd to me. Does "retrieves" fit this better?

about the ``sample_restaurants`` database: 

.. include:: /includes/usage-examples/run-example-tip.rst

.. literalinclude:: /includes/usage-examples/code-snippets/command.go
   :start-after: begin runCommand
   :end-before: end runCommand
   :emphasize-lines: 5
   :language: go
   :dedent:

Click `here <{+example+}/command.go>`__ to see a fully runnable example.

Expected Result
---------------

After running the preceding code snippet, the ``result`` variable should

> contain the following: 

Reviewer: ```suggestion
contain the following cursor (maybe struct?): 
```
Suggestion: Could this be more specific? I am not sure what I am looking at, and needed to consult API docs to try and figure it out.

.. code-block:: json

    // results truncated
    [ {db sample_restaurants} 
      {collections 2} 
      {views 0} 
      {objects 25554} 
      {avgObjSize 548.4101901854896} 
      {dataSize 14014074} {storageSize 8257536} 
      ... ]

.. note::

>     The ``result`` variable may vary depending on your data set.

Reviewer: ```suggestion
    The ``result`` variable may vary depending the contents of your collection.
```
Nit: This struck me as odd as the data set we are providing is in theory the same for everyone. I think this has more to do with the readers "collection" (specific of what is on their machine right now) than their "data set" (which in my head is the thing they downloaded from atlas the first time).
